[DOCS] Adds the 8.6.2 release notes (#151028)
elastic/kibana: docs/CHANGELOG.asciidoc

There are no breaking changes in {kib} 8.6.2.
// end::notable-breaking-changes[]

{kibana-ref-all}/8.5/release-notes-8.5.0.html#breaking-changes-8.5.0[8.5.0] | {kibana-ref-all}/8.4/release-notes-8.4.0.html#breaking-changes-8.4.0[8.4.0] | {kibana-ref-all}/8.3/release-notes-8.3.0.html#breaking-changes-8.3.0[8.3.0] | {kibana-ref-all}/8.2/release-notes-8.2.0.html#breaking-changes-8.2.0[8.2.0] | {kibana-ref-all}/8.1/release-notes-8.1.0.html#breaking-changes-8.1.0[8.1.0] | {kibana-ref-all}/8.0/release-notes-8.0.0.html#breaking-changes-8.0.0[8.0.0] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc2.html#breaking-changes-8.0.0-rc2[8.0.0-rc2] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc1.html#breaking-changes-8.0.0-rc1[8.0.0-rc1] | {kibana-ref-all}/8.0/release-notes-8.0.0-beta1.html#breaking-changes-8.0.0-beta1[8.0.0-beta1] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha2.html#breaking-changes-8.0.0-alpha2[8.0.0-alpha2] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha1.html#breaking-changes-8.0.0-alpha1[8.0.0-alpha1]

[float]
[[enhancement-v8.6.2]]
=== Enhancements
Alerting::
Adds the open API specification for create rule {kibana-pull}149565[#149565]

Fleet::

> Adds the ability to run agent policy schema in batches during fleet setup + add `xpack.fleet.setup.agentPolicySchemaUpgradeBatchSize` config {kibana-pull}150688[#150688]

Reviewer: ```suggestion
Adds the ability to run agent policy schema in batches during fleet setup. Also adds `xpack.fleet.setup.agentPolicySchemaUpgradeBatchSize` config {kibana-pull}150688[#150688]
```

Security::

> Adds support for handlebars template "inline partials" when `csp.disableUnsafeEval` is set to `true`, allowing you to use inline partials to remove the `unsafe-eval` source expression in the {kib} Content Security Policy (CSP) {kibana-pull}150151[#150151]

Reviewer 2: ```suggestion
Adds support for handlebars template "inline partials" without requiring `unsafe-eval` source expression in the {kib} Content Security Policy (CSP) {kibana-pull}150151[#150151]
```
